osbs/build/build_request.py

```diff
@@ -1171,7 +1171,7 @@ def render_pulp_sync(self):
 
             source_registry = self.spec.source_registry_uri.value
             perform_delete = (source_registry is None or
-                              source_registry.docker_uri != registry.docker_uri)
```

Comment:
It should be of `RegistryURI` type: https://github.com/projectatomic/osbs-client/blob/master/osbs/build/spec.py#L114

Reply:
yeah I've just find that out ..